values.yaml

```diff
@@ -138,6 +138,12 @@ internalTLS:
   certSource: "auto"
   # The content of trust ca, only available when `certSource` is "manual"
   trustCa: ""
+  certValidity:
+    # Number of days self-signed certificate is valid
+    caCert: 365
+    # CA certificate validity in days
```

Comment:
the comment of line 144 should be placed before line 143.
the comment of line 146 should be placed before line 145